default/scripting/species/SP_KHAKTURIAN.focs.txt

```diff
@@ -0,0 +1,72 @@
+Species
+    name = "SP_KHAKTURIAN"
+    description = "SP_KHAKTURIAN_DESC"
+    gameplay_description = "SP_KHAKTURIAN_GAMEPLAY_DESC"
+    Native
+    CanProduceShips
+
+    // Desert: Khakturian: Planetbound, Great Attack Troops, Great Defense Troops, Phototrophic
+
+    tags = [ "PHOTOTROPHIC" "BAD_POPULATION" "BAD_INDUSTRY" "NO_RESEARCH" "BAD_INFLUENCE" "GOOD_STOCKPILE" "GREAT_ATTACKTROOPS" "PEDIA_PHOTOTROPHIC_SPECIES_CLASS" ]
+
+    foci = [
+        [[HAS_INDUSTRY_FOCUS]]
+        /*[[HAS_RESEARCH_FOCUS]]*/    // dumb plants after all
+        /*[[HAS_INFLUENCE_FOCUS]]*/   // see above
+        [[HAS_GROWTH_FOCUS]]
+        [[HAS_ADVANCED_FOCI]]
+    ]
+    defaultfocus = "FOCUS_GROWTH"
+
+    // i'll put comments for motivation on likes for later textification into pedia
+    likes = [
+        "BLD_LIGHTHOUSE"          // if there were a greenhouse they would like that too
+        "FOCUS_GROWTH"            // what else would a plant want?
+        "FOCUS_PROTECTION"        // yep, that one too.. happily bristling with thorns, soaking sun, la la la
+        "BLD_STOCKPILING_CENTER"  // why, to store all the seeds for the next rainy season of course
+        "TIDAL_LOCK_SPECIAL"      // eternal daylight yeah! nightsiders? not a problem for us prickly plants we got no blood
+        "HEAD_ON_A_SPIKE_SPECIAL" // we likes the spikes
+        "PLC_POPULATION"          // grow, grow, grow your load, gently down the ..
+    ]
+    dislikes = [ // dislikes all other desert species because they compete for habitat
```

Comment:
no, we do not want that "dislikes all * in the same environment". I think having the same preferred environment is not strong enough to warrant the inflation of dis/likes - at least this needs a thorough discussion (either on dev meeting or forums). also the other desert species probably do not compete for the same niche. dislike gameplay wise means it causes the population unrest, to revolt, stop working, want to become independent etc. e.g. presence of nightpsiders terrifies - that makes much sense.
the worldtree could make sense dislike wise if it causes distress to khakturians, but the fluff just sais it is a single gigantic tree. i would imagine the khakturians not to care about it. 
clouds/volcanic ash dislike makes much sense fluff-wise. gameplay wise it is slightly smelly, as there is a design rule for techs which is: techs are not allowed to make situation worse for the player (as there is no way to take a tech back). One could argument that you can remove those by researching the third stealth tech, but it is a stretch.

Reply:
i can easily comment those out since it was already noted by geoff that (dis)liking species has no effect currently other than showing up in the pedia page (where it fits their stories is why i put them there) and we can discuss at leasure in the forum whether to un-uncomment them again at a later date

Reply:
aye

Reply:
and done.

Reply:
@agrrr3 - i only just now noticed that the world tree is disliked by many more species, among them laenfa who are also plants. should i do something about that too?
![worldtree more disliked than liked](https://user-images.githubusercontent.com/37298400/151089075-e33e735b-de5d-495f-9862-4fbd48bf8b86.png)